tool_nodes/dora-rerun/src/main.rs

```diff
@@ -21,87 +21,94 @@ fn main() -> Result<()> {
     let mut options = SpawnOptions::default();
 
     let memory_limit = std::env::var("RERUN_MEMORY_LIMIT")
-        .context("Could not read image height")?
+        .context("Could not read image height")
```

Comment:
The context message doesn't really make sense here (image height?).

Reply:
Indeed sorry!